### [Microsoft.FactoryOrchestrator.Core](Microsoft_FactoryOrchestrator_Core.md 'Microsoft.FactoryOrchestrator.Core').[FactoryOrchestratorException](FactoryOrchestratorException.md 'Microsoft.FactoryOrchestrator.Core.FactoryOrchestratorException')
## FactoryOrchestratorException.FactoryOrchestratorException(string, Exception) Constructor
Initializes a new instance of the [FactoryOrchestratorException](FactoryOrchestratorException.md 'Microsoft.FactoryOrchestrator.Core.FactoryOrchestratorException') class.  
```csharp
public FactoryOrchestratorException(string message, System.Exception innerException);
```
#### Parameters
<a name='Microsoft_FactoryOrchestrator_Core_FactoryOrchestratorException_FactoryOrchestratorException(string_System_Exception)_message'></a>
`message` [System.String](https://docs.microsoft.com/en-us/dotnet/api/System.String 'System.String')  
The error message that explains the reason for the exception.
  
<a name='Microsoft_FactoryOrchestrator_Core_FactoryOrchestratorException_FactoryOrchestratorException(string_System_Exception)_innerException'></a>
`innerException` [System.Exception](https://docs.microsoft.com/en-us/dotnet/api/System.Exception 'System.Exception')  
The exception that is the cause of the current exception, or a null reference (Nothing in Visual Basic) if no inner exception is specified.
  
